src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -193,54 +160,36 @@ const overviewOrgSlice = createSlice({
       }
     },
     save: (state) => {
-      const list = mapToSoftDeletedOrgDto(state);
-      console.log(list);
+      console.log(state.overviewOrgs);
     },
-    softUndeleteAll: (state, action) => {
-      const undeletedOrgId = action.payload;
-      for (const org of state.softDeletedOverviewOrgs) {
-        if (org.id === undeletedOrgId) {
-          removeSoftDeletedOrgReference(state, undeletedOrgId);
-          break;
-        }
-      }
+    softRestoreAll: (state, action) => {
+      const restoredOrgId = action.payload;
 
-      setSoftDeleteState(state, undeletedOrgId, false);
+      setAllItemsToGivenSoftDeleteState(state, restoredOrgId, false);
     },
     softDeleteAll: (state, action) => {
       const selectedOrgId = action.payload;
-      for (const softDeletedOrg of state.softDeletedOverviewOrgs) {
-        if (softDeletedOrg.id === selectedOrgId) {
-          removeSoftDeletedOrgReference(state, selectedOrgId);
-        }
-      }
-      state.softDeletedOverviewOrgs.push(selectedOrgId);
 
-      setSoftDeleteState(state, selectedOrgId, true);
-    },
-    setIsEditable: (state, action) => {
-      state.overviewOrgIsEditable = action.payload;
+      setAllItemsToGivenSoftDeleteState(state, selectedOrgId, true);
     },
-    emptySoftDeletedList: (state) => {
+    emptySoftDeletedItems: (state) => {
```

Comment:
Suggested namechange: 
restoreAllSoftDeletedItems

You aren't really emptying anything anymore